python/lsst/dax/imgserv/locateImage.py

```diff
@@ -400,12 +417,12 @@ def _getImageButler(self, qResults):
     def _getMetadata(self, butler, qResults):
         '''Return the metadata for the query results in qResults and a butler
         '''
-        for ln in qResults:
-            tract = ln[2]
-            patch = ln[3]
-            filterName = ln[4]
+        valid, tract, patch, filterName = _getKeysForButler2(qResults)
+        if valid is True:
             metadata = butler.get(self.getImageDatasetMd(), tract=tract, patch=patch, filter=filterName)
             return metadata
+        else:
```

Comment:
Removed all "return None" that are the last statements of their functions, in absence of any guidance from the DM Python Style Guide.